tensorflow_quantum/python/util.py

```diff
@@ -467,6 +489,16 @@ def _expression_approx_eq(exp_1, exp_2, atol):
     return False
 
 
+# TODO: replace with cirq.approx_eq once
+# https://github.com/quantumlib/Cirq/issues/3886 is resolved for all channels.
+def _channel_approx_eq(op_true, op_deser, atol=1e-5):
+    if isinstance(op_true, cirq.DepolarizingChannel):
+        if isinstance(op_deser, cirq.DepolarizingChannel):
+            return abs(op_true.p - op_deser.p) < atol
+
+    return False
```

Comment:
nit:`raise TypeError("op_true and op_deser should be of cirq.DepolorizingChannel"`, to distinguish the right False and wrong type cases.

Reply:
In this case if a Channel is supplied that isn't supported, you will still raise the `raise ValueError(
        f"Some valid TFQ gate type is not yet accounted for, got {gate_true}")` at the end. Right now there is no support for other channels so it will actually behave correctly if an unknown channel is supplied.